scripts/create-cf-stack.py

```diff
@@ -113,6 +113,7 @@ def run_with_pty(self, command):
     (ssh_key_name, ssh_key) = valid_config["ssh"][REGION]
     ec2_key = valid_config["ec2"]["ec2-key"]
     ec2_secret_key = valid_config["ec2"]["ec2-secret-key"]
+    ec2_name = ssh_key_name.split('-', 1)[0]
```

Comment:
@pbartiko, I have just run it and now I have instances named like
igulina_eu_RHEL7_nfs_20170118_nfs
igulina_eu_RHEL7_nfs_20170118_haproxy1, etc

'eu' doesn't need to be there. It's because my ssh key file is named 'igulina_eu-west1.pem', i.e. split fails if one has ssh_key_name different from pattern '$whatever_before_slash' + '-' + '$whatever_after_slash'.

Line #116 should be replaced with:
ec2_name=re.search("[a-zA-Z]+", ssh_key_name).group(0)
and 'import re' added respectively. Thanks.